Library/Homebrew/utils/puts_columns.rb

```diff
@@ -0,0 +1,41 @@
+require "utils/tty"
+
+module Kernel
```

Comment:
We could change it to only format the columns, and not output them, and let default `puts` to the rest.

```ruby
puts Formatter.columns(something)
```